pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Operations Features
```

Comment:
The document is about operations alone so it would give out the same meaning. Again, Keep the headings as concise yet informative as possible.
```suggestion
## Features
```